packages/fuel-gauge/src/launch-test-node-types.test.ts

```diff
@@ -1,29 +0,0 @@
-import { launchTestNode } from 'fuels/test-utils';
-
-import type { AdvancedLoggingAbi, CallTestContractAbi } from '../test/typegen';
-import { AdvancedLoggingAbi__factory, CallTestContractAbi__factory } from '../test/typegen';
-import AdvancedLoggingBytecode from '../test/typegen/contracts/AdvancedLoggingAbi.hex';
-import CallTestContractBytecode from '../test/typegen/contracts/CallTestContractAbi.hex';
-
-/**
- * @group node
- * @group browser
- */
-describe('type level tests for launchTestNode', () => {
```

Comment:
This test currently doesn't add any value as type inferencing is done statically

Reply:
If I understood @nedsalk correctly, the value is that it would catch type errors that could potentially go unnoticed when running the tests via Vitest, which can sometimes run tests even if they contain type mismatches.

Reply:
I didn't manage to get this working as expected with `vitest` so I'm okay with removing it. I created #2775 to revisit this.